environments/auth/package.json

```diff
@@ -3,6 +3,8 @@
   "name": "environments-auth",
   "version": "0.0.1",
   "scripts": {
-    "pull": "../pull-environments.sh"
+    "pull": "../pull-environments.sh -r us-east-1",
+    "pull:phone": "../pull-environments.sh -r us-east-1 -i \"\\./auth-with-((phone-and-sms-mfa)|(phone-number)|(totp-and-sms-mfa))\"",
```

Comment:
While this is slightly ugly with escapings, this lets us filter just the sms/phone number environments.